hw/dv/tools/dvsim/sim.mk

```diff
@@ -97,9 +94,24 @@ ifneq (${sw_images},)
 			fi; \
 			echo "Building with command: $${bazel_cmd} build $${bazel_opts} $${bazel_label}"; \
 			$${bazel_cmd} build $${bazel_opts} $${bazel_label}; \
-			find -L $$($${bazel_cmd} info output_path)/ \
-				-type f -name "$${bazel_target}*" | \
-				xargs -I % sh -c 'cp -f % $${run_dir}/$$(basename %)'; \
+			for dep in $$($${bazel_cmd} cquery "labels(data, $${bazel_label})" \
+				--ui_event_filters=-info \
```

Comment:
nit: sorry, could you replace tabs with 2-spaces instead?

Reply:
I think that's a bad idea for maintainability because of [how Makefiles work](https://stackoverflow.com/questions/28712585/when-to-use-space-or-tab-in-makefile) and how editors (e.g., my VIM config) are often configured to use tabs in Makefiles. Specifically, for Make to know what is part of the recipe, the line must begin with a tab (not spaces). So essentially we would be maintaining two separate tabbing styles across the file (which I think most editors are not going to do automatically; if there's a VIM plugin tho lmk!). Moreover, I don't believe we have a style guide for makefiles, if we do I definitely missed it.

Reply:
IIUC only the first char needs to be a tab. It's ok - not a big deal.

Reply:
Yeah that's exactly right, and because of that its hard for editors to automatically switch from expanding tabs to spaces and keeping them as tabs.